.github/workflows/php.yml

```diff
@@ -56,13 +56,13 @@ jobs:
             matrix:
                 wp: ['latest']
                 wpmu: [0]
-                php: ['7.0', '7.1', '7.2', '7.3', '7.4', '8.0']
+                php: ['7.2', '7.3', '7.4', '8.0']
```

Comment:
Since this PR is only for a recommendation, should we still keep testing against 7.0 and just update it when we don't support it anymore?

Reply:
Ah, actually, since it's for the next release and it will be in 2 days, I think that's fine to already remove it. Sorry for the noise!